presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1288,6 +1271,30 @@ private void testInsertPartitionedBucketedTableWithUnionAll(HiveStorageFormat st
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testInsertTwiceToSamePartitionedBucket()
+    {
+        String tableName = "test_duplicated_insert_with_partitioned_bucketed_table";
+        createPartitionedBucketedTable(tableName, HiveStorageFormat.RCBINARY);
+
+        String insertQuery = format("INSERT INTO " + tableName + " " +
+                "SELECT custkey, custkey AS custkey2, comment, orderstatus " +
+                "FROM tpch.tiny.orders " +
+                "WHERE orderstatus = '%s'", "F");
+        assertUpdate(
+                insertQuery,
+                format("SELECT count(*) from orders where orderstatus = '%s'", "F")
+        );
```

Comment:
per code style, put the `);` on the previous line;

also the whole `assertUpdate(insertQuery, ...)` statement is short enough so you can put it on one line

Reply:
travis says:
```
INFO] There are 2 errors reported by Checkstyle 8.16 with checkstyle/airbase-checks.xml ruleset.
[ERROR] src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java:[1287] (regexp) RegexpMultiline: Whitespace character before closing parenthesis
[ERROR] src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java:[1292] (regexp) RegexpMultiline: Whitespace character before closing 
```